docs/components/store.md

```diff
@@ -308,10 +310,24 @@ While the remaining settings are **optional**:
 
 ### Redis index cache
 
-The `redis` index cache allows to use [Redis](https://redis.io) as cache backend. This cache type is configured using `--index-cache.config-file` to reference the configuration file or `--index-cache.config` to put yaml config directly:
+The `redis` index cache allows to use [Redis](https://redis.io) as cache backend. This cache type is configured using `--index-cache.config-file` to reference the configuration file or `--index-cache.config` to put YAML config directly.
+
+It supports two types of clients:
+
+* `GOREDIS` (the default one)
+* `RUEIDIS`
+
+The main difference is that [Rueidis](https://github.com/rueian/rueidis) supports [client-side caching](https://redis.io/docs/manual/client-side-caching/) which is really important. If you have high cardinality metrics then it means that it is no longer necessary to go to remote-object storage to get information about potentially thousands or millions of metrics.
```

Comment:
> If you have high cardinality metrics then it means that it is no longer necessary to go to remote-object storage to get information about potentially thousands or millions of metrics.

I think this is wrong. The client caching is for Redis client NOT object storage. Also how it helps with cardinality in particular? 

Client side caching to me sounds like extra in-mem cache that is checked first before accessing shared cache in REDIS, right?

Reply:
Yeah I am also a little bit confused. What's the benefits of client side caching over server side caching. Less network calls?